integ-test/src/test/java/org/opensearch/sql/legacy/AggregationIT.java

```diff
@@ -556,42 +597,58 @@ public void countGroupByRange() throws IOException {
-            "\"min_doc_count\":5}"));
+        explainQuery(
+            String.format(
+                "select insert_time from %s group by"
+                    + " date_histogram('field'='insert_time','fixed_interval'='1h','format'='yyyy-MM','min_doc_count'=5)"
+                    + " ",
+                TEST_INDEX_ONLINE));
+    Assert.assertThat(
+        result.replaceAll("\\s+", ""),
+        containsString(
+            "{\"date_histogram\":{\"field\":\"insert_time\",\"format\":\"yyyy-MM\","
+                + "\"fixed_interval\":\"1h\",\"offset\":0,\"order\":{\"_key\":\"asc\"},\"keyed\":false,"
+                + "\"min_doc_count\":5}"));
   }
 
   @Test
   public void countGroupByDateTestWithAlias() throws IOException {
     String result =
-        explainQuery(String.format("select insert_time from %s group by date_histogram" +
-                "('field'='insert_time','fixed_interval'='1h','format'='yyyy-MM','alias'='myAlias')",
-            TEST_INDEX_ONLINE));
-    Assert.assertThat(result.replaceAll("\\s+", ""),
-        containsString("myAlias\":{\"date_histogram\":{\"field\":\"insert_time\"," +
-            "\"format\":\"yyyy-MM\",\"fixed_interval\":\"1h\""));
-  }
-
-//    /**
-//     * <a>http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/search-aggregations-bucket-daterange-aggregation.html</a>
-//     */
-//    @Test
-//    public void countDateRangeTest() throws IOException, SqlParseException, SQLFeatureNotSupportedException {
-//        String result = explainQuery(String.format("select online from %s group by date_range(field='insert_time'," +
-//                "'format'='yyyy-MM-dd' ,'2014-08-18','2014-08-17','now-8d','now-7d','now-6d','now')",
-//                TEST_INDEX_ONLINE));
-//        // TODO: fix the query or fix the code for the query to work
-//    }
+        explainQuery(
+            String.format(
+                "select insert_time from %s group by date_histogram"
+                    + "('field'='insert_time','fixed_interval'='1h','format'='yyyy-MM','alias'='myAlias')",
+                TEST_INDEX_ONLINE));
+    Assert.assertThat(
+        result.replaceAll("\\s+", ""),
+        containsString(
+            "myAlias\":{\"date_histogram\":{\"field\":\"insert_time\","
+                + "\"format\":\"yyyy-MM\",\"fixed_interval\":\"1h\""));
+  }
+
+  /**
+   * <a>
+   * http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/search-aggregations-bucket-daterange-aggregation.html</a>
+   */
+  @Test
+  @Ignore
+  public void countDateRangeTest() throws IOException {
+    String result =
+        explainQuery(
+            String.format(
+                "select online from %s group by date_range("
+                    + "field='insert_time', 'format'='yyyy-MM-dd' ,'2014-08-18','2014-08-17', "
+                    + "'now-8d','now-7d','now-6d','now')",
+                TEST_INDEX_ONLINE));
+    // TODO: fix the query or fix the code for the query to work
+  }
```

Comment:
Should this have been uncommented out?

Reply:
Was requested here: https://github.com/Bit-Quill/opensearch-project-sql/pull/327#discussion_r1284888525. The @ignore was added to not run the test.